### DISCUSSION TITLE
数据库连接端口设置

### DISCUSSION THREAD

**Discussion by Ludovicofu** [antoor/antSword#332] [Q&A] [answered]
比如mysql默认端口不是3306改成了其他端口，使用蚁剑的数据库管理功能无法设置改变连接端口

**Comment by Medicean** [comment]
哪个shell 类型,用的 `mysql` 还是 `mysqli` ?

**Comment by Ludovicofu** [comment]
php，mysql

**Comment by Medicean** [comment, marked as answer]
@Ludovicofu 测试正常，你要不看一下配置是否和我的一样?

![image](https://user-images.githubusercontent.com/6082020/218696748-cbf8e1ef-1af6-4258-b980-a62fb1ac23f6.png)

![image](https://user-images.githubusercontent.com/6082020/218696694-f24d2e13-30ac-4ff9-88da-764104ebc88a.png)

**Comment by Ludovicofu** [comment]
好吧，搞错了
